File: ChatkitEmojiAndroidApp/app/src/main/java/com/neo/chatkitemoji/MessageModel.kt
package com.neo.chatkitemoji

import com.pusher.chatkit.messages.Message

data class MessageModel(val message: Message, val data:ArrayList<EmojiModel>)

data class EmojiModel (var string :String = "",
                       var count :Int = 1,
                       var userIds :ArrayList<String> = ArrayList())

File: ChatkitEmojiIOSApp/ChatkitEmojiTest/MessageModel.swift
import PusherChatkit

class MessageModel {
    
    var message:PCMessage!
    var emojiModelList = [EmojiModel]()
    
}

class EmojiModel {
    var string:String?
    var count: Int?
    var usersToEmoji: [String]?
}
